client/packages/android/app/src/main/java/org/openmsupply/client/CertWebViewClient.java

```diff
@@ -1,34 +1,39 @@
-package org.openmsupply.client.certplugin;
+package org.openmsupply.client;
```

Comment:
In this file, main changes were splitting validation to `validateNonLocalCertificate` and `validateLocalCertificate` other changes should be very similar to changes in `electront.ts`. We also save nativeAPI in constructor (to access connectedServer)